src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -83,7 +82,7 @@ public CursorsController(final CursorsService cursorsService,
         try {
             final List<SubscriptionCursor> cursors = cursorsService.getSubscriptionCursors(subscriptionId, client)
                     .stream()
-                    .map(cursor -> cursorConverter.convert(cursor, cursorTokenService.generateToken()))
+                    .map(cursor -> cursor.withToken(cursorTokenService.generateToken()))
```

Comment:
@antban 
This is clear, I may read it even without ide (in github pr):
`map(noTokenCursor -> SubscriptionCursor.from(noTokenCursor, cursorTokenService.generateToken())`
Yes, it is longer by 14 chars.

@lmontrieux not the name what bothers me, but how it is organised structural.